Wrappers/Python/cil/framework/framework.py

```diff
@@ -721,60 +736,44 @@ def update_reference_frame(self):
         new_y = rotation_matrix.dot(self.detector.direction_y.reshape(3,1))
         self.detector.set_direction(new_x, new_y)
 
-    def align_reference_frame(self):
-        r'''Transforms the system origin to the rotate axis with z direction aligned to the rotate axis direction, and aligns the ray direction along the positive Y direction
-        '''          
-        self.update_reference_frame()
-
-        ray_vec = -self.ray.direction
-        axis_rotation = numpy.eye(3)
 
-        if numpy.allclose(ray_vec,[0,-1,0]):
-            pass
-        elif numpy.allclose(ray_vec,[0,1,0]):
-            axis_rotation[0][0] = -1
-            axis_rotation[1][1] = -1
-        else:
-            theta = math.atan2(ray_vec[0],ray_vec[1])
-            axis_rotation[0][0] = axis_rotation[1][1] = math.cos(theta)
-            axis_rotation[0][1] = -math.sin(theta)
-            axis_rotation[1][0] = math.sin(theta)
-
-        rotation_matrix = numpy.matrix(axis_rotation)
-        
+    def align_reference_frame(self, definition='cil'):
```

Comment:
where is `definition` used?

Reply:
In this instance the behaviour whether 'cil' or 'tigre' definitions are the same. I've added a check on the input value, and added a comment to explain why it's seemingly irrelevant here (although used by other versions of the function)